rocksdb_replicator/thrift/replicator.thrift

```diff
@@ -56,7 +59,8 @@ struct Update {
 enum ReplicaRole {
   NOOP = 0, // no replication needed
   FOLLOWER = 1,
-  LEADER = 2
+  LEADER = 2,
+  OBSERVER = 3 // replicates but should not be counted as a follower ack
```

Comment:
can you update https://github.com/pinterest/rocksplicator/blob/master/rocksdb_replicator/utils.h#L21 as well